src/content/community/grants/index.md

```diff
@@ -16,7 +16,7 @@ This list is curated by our community. If there's something missing or incorrect
 These programs support the broad Ethereum ecosystem by offering grants to a wide scope of projects. These include solutions for scalability, community building, security, privacy and more. These grants are not specific to any one Ethereum platform, and are a good place to start if you're unsure.
 
 - [EF Ecosystem Support Program](https://esp.ethereum.foundation) - _Funding open source projects that benefit Ethereum, with a particular focus on universal tools, infrastructure, research and public goods_
-- [EF Rollup Community Grants Wave](https://esp.ethereum.foundation/en/rollup-grants/) - _For everything related to rollups. Proposals are due April 16th 2021_
+- [Ethereum RFPs](https://github.com/ethereum/requests-for-proposals) - _Requests for Proposals for work and projects in the Ethereum ecosystem_
```

Comment:
Hey @ethosdev thanks for the PR.

> _Requests for Proposals for work and projects in the Ethereum ecosystem_

I don't think the wording here is super clear. Would it be correct to say:

> Request for Proposals by the Ethereum Foundation for work and projects in the Ethereum ecosystem?

Reply:
Sure I added the clarification. Thanks

Reply:
I think part of the problem is that the README in the RFPs repo is pretty bare and confusing to those without context. I've left a PR saying as much.

Either way, merging now! Thanks again. ✌️